tests/comfy_cli/command/test_workflow_edit_promoted.py

```
@@ -0,0 +1,307 @@
+    link = next(link for link in wf["links"] if link[0] == op["link_id"])
+    assert link[1] == prim and link[3] == 57
+
+
+def test_connect_type_mismatch_into_a_promoted_input_is_refused(graph):
+    wf = _load("image_z_image_turbo.json")
+    wf, text = workflow_ops.add_node(wf, graph, "PrimitiveStringMultiline")
+    with pytest.raises(ValueError, match="type mismatch"):
+        workflow_ops.connect(wf, graph, text["node_id"], "STRING", 57, "width")
+
+
+def test_connect_to_an_already_materialized_promoted_input_reuses_it(graph):
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)